modules/light-clients/08-wasm/types/client_state_test.go

```diff
@@ -109,53 +113,44 @@ func (suite *TypesTestSuite) TestStatusTendermint() {
 		{
 			"client is frozen",
 			func() {
-				tmClientState.FrozenHeight = clienttypes.NewHeight(0, 1)
-
-				wasmData, err := suite.chainA.Codec.MarshalInterface(tmClientState)
-				suite.Require().NoError(err)
-
-				clientState.Data = wasmData
-				path.EndpointA.SetClientState(clientState)
+				suite.mockVM.QueryFn = func(codeID wasmvm.Checksum, env wasmvmtypes.Env, queryMsg []byte, store wasmvm.KVStore, goapi wasmvm.GoAPI, querier wasmvm.Querier, gasMeter wasmvm.GasMeter, gasLimit uint64, deserCost wasmvmtypes.UFraction) ([]byte, uint64, error) {
+					resp := fmt.Sprintf(`{"status":"%s"}`, exported.Frozen)
+					return []byte(resp), wasmtesting.DefaultGasUsed, nil
+				}
 			},
 			exported.Frozen,
 		},
 		{
-			"client status without consensus state",
+			"client status is expired",
 			func() {
-				latestHeight := clientState.LatestHeight.Increment().(clienttypes.Height)
-				tmClientState.LatestHeight = latestHeight
-
-				wasmData, err := suite.chainA.Codec.MarshalInterface(tmClientState)
-				suite.Require().NoError(err)
-
-				clientState.Data = wasmData
-				clientState.LatestHeight = latestHeight
-				path.EndpointA.SetClientState(clientState)
+				suite.mockVM.QueryFn = func(codeID wasmvm.Checksum, env wasmvmtypes.Env, queryMsg []byte, store wasmvm.KVStore, goapi wasmvm.GoAPI, querier wasmvm.Querier, gasMeter wasmvm.GasMeter, gasLimit uint64, deserCost wasmvmtypes.UFraction) ([]byte, uint64, error) {
+					resp := fmt.Sprintf(`{"status":"%s"}`, exported.Expired)
```

Comment:
can also do something like:
```go
        statusResult := new(types.StatusResult)
        statusResult.Status = exported.Active
        resp, err := json.Marshal(&statusResult)
        suite.Require().NoError(err)
```
I just chose something

Reply:
I actually thing the one liner you have is super readable. I think the other approach you suggested would be preferable for larger types. 

That, or pass an fn that acts on the type and handle the marshalling in separately.